charts/merlin/values.yaml

```diff
@@ -1,11 +1,16 @@
 global:
   protocol: http
+
+# rendered takes precedence over fields defined in config
+# set to empty dictionary to use values in config
```

Comment:
Because of this line here: https://github.com/caraml-dev/helm-charts/pull/351/files#diff-c4a8d768b754993bd138320762c1283409292bc538392f0976e85a246c8231bdR311,
```yaml
{{- $tag :=  ternary $.Values.deployment.image.tag $rendered.releasedVersion (ne $.Values.deployment.image.tag "") -}}
```
shouldn't the comment read, "set deployment.image.tag to non-nil to use values in config"? i.e. if I have both `rendered.releasedVersion` and `deployment.image.tag` set, the tag used is the latter right? 🤔

Reply:
Yes the tag used will be the one in `deployment.image.tag`

Reply:
What i meant to say was the `.rendered` dictionary will override fields in `.config` here https://github.com/caraml-dev/helm-charts/blob/97996550a3d2c4d04ffd511423f35e7c26ee8700/charts/merlin/values.yaml#L83

Will rephrase it to describe it better

Reply:
Ahh okay, got it 😅 Thanks for clarifying this!